apps/api/src/modules/user/user.service.ts

```diff
@@ -29,4 +34,31 @@ export class UserService {
       }
     return userData
   }
+
+  async getPendingUsers(): Promise<PendingUserDto[]> {
+    const users = await this.repository.getPendingUsers({})
+    return users
+  }
+  async updateUserStatus(
+    userId: number,
+    updateUserStatusDto: UpdateUserStatusDto,
+  ): Promise<GetUserDto> {
+    //check if user exist
+    const user = await this.repository.getUserById(userId)
+    if (!user) throw new NotFoundException()
+
```

Comment:
@miello Do we need to check if user can be accept or reject only if their status is "Pending"

Reply:
It should

Reply:
It should

Reply:
But there is interesting case that admin wants to change it back so in this case it might need not to check.

Reply:
I think we can ช่างแม่ง in this case

Reply:
งั้นเอาแค่ปรับจาก PENDING เป็นอย่างอื่นละกัน ง่ายดี ;w;